### ISSUE TITLE
[BUG]Draconic Crafting injectors not crafting

### ISSUE THREAD

**Issue by tumdug** [brandon3055/Draconic-Evolution#1151]
## Base information
* Minecraft version: 1.12.2
* Mod version: Draconic-Evolution-1.12.2-2.3.12.297-universal.jar
* Minecraft Forge version: Forge-14.23.4.2715
* Mod Pack: (if applicable)
ATM3 Version- 5.9.8d

Is NOT a crash report!</br>
![image](https://user-images.githubusercontent.com/42040826/43595243-cf18386e-9641-11e8-9f9b-ea74ebdf5571.png)
I have the correct recipe to craft the draconic crafting injector but clicking the start button doesn't do anything
I have power to the injectors since crafting other things like wyvern cores work but not this recipe.
![image](https://user-images.githubusercontent.com/42040826/43595333-ffd180be-9641-11e8-91ad-14bd9a8454f7.png)

**Comment by GenghisKahn** [comment]
I'm having the same issue. 

- Minecraft version: 1.12.2
- Mod version: Draconic-Evolution-1.12.2-2.3.12.297-universal.jar
- Minecraft Forge version: Forge-14.23.4.2705
- Mod Pack: (if applicable) Lapito's Galacticraft v1.7

I'm unable to upgrade from Wyvren Tier to Draconic tier injectors.

Recipe and start button show up but clicking start does nothing.
![2018-09-07_22 22 21](https://user-images.githubusercontent.com/7990601/45249411-c06a8600-b2ed-11e8-8c3b-17e930048179.png)

I've tried both 1 block between and 2 blocks between the injectors and the core.
![2018-09-07_22 22 32](https://user-images.githubusercontent.com/7990601/45249419-e09a4500-b2ed-11e8-84ed-861ebe0a800a.png)


Other recipes work fine as far as I can tell. I tested with Draconic chest and awakened Draconium.
